ui/src/app/app.tsx

```diff
@@ -110,7 +98,17 @@ export class App extends React.Component<{}, {version?: Version; popupProps: Pop
 
     public componentDidMount() {
         this.popupManager.popupProps.subscribe(popupProps => this.setState({popupProps}));
-        services.info.getVersion().then(version => this.setState({version}));
+        services.info
+            .getVersion()
+            .then(version => this.setState({version}))
+            .then(() => services.info.getInfo())
+            .then(info => this.setState({namespace: info.managedNamespace || Utils.getCurrentNamespace() || ''}))
+            .catch(error => {
```

Comment:
I make an exception here - report this error in a popup. We still need to be able to get to the login page on 401.